### ISSUE TITLE
java.lang.ClassNotFoundException: net.minecraft.server.v1_8_R2.SlotAnvilResult

### ISSUE THREAD

**Issue by mibby** [Siguza/VirtualPack#28]
@Siguza 
VirtualPack dev 132
Spigot 1.8.3 (git-Spigot-ea179b3-6e0120a)

```
[23:55:42] [Server thread/INFO]: mibby issued server command: /anvil
[23:55:42] [Server thread/WARN]: java.lang.ClassNotFoundException: net.minecraft.server.v1_8_R2.SlotAnvilResult
[23:55:42] [Server thread/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:91)
[23:55:42] [Server thread/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:86)
[23:55:42] [Server thread/WARN]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[23:55:42] [Server thread/WARN]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[23:55:42] [Server thread/WARN]: 	at java.lang.Class.forName0(Native Method)
[23:55:42] [Server thread/WARN]: 	at java.lang.Class.forName(Class.java:264)
[23:55:42] [Server thread/WARN]: 	at net.drgnome.virtualpack.components.VAnvil.<init>(VAnvil.java:30)
[23:55:42] [Server thread/WARN]: 	at net.drgnome.virtualpack.VPack.openAnvil(VPack.java:1295)
[23:55:42] [Server thread/WARN]: 	at net.drgnome.virtualpack.VCommands.anvil(VCommands.java:1623)
[23:55:42] [Server thread/WARN]: 	at net.drgnome.virtualpack.VCommands.tools(VCommands.java:183)
[23:55:42] [Server thread/WARN]: 	at net.drgnome.virtualpack.VCommands.cmd(VCommands.java:120)
[23:55:42] [Server thread/WARN]: 	at net.drgnome.virtualpack.VCommands.onCommand(VCommands.java:77)
[23:55:42] [Server thread/WARN]: 	at com.sk89q.bukkit.util.DynamicPluginCommand.execute(DynamicPluginCommand.java:54)
[23:55:42] [Server thread/WARN]: 	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141)
[23:55:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_8_R2.CraftServer.dispatchCommand(CraftServer.java:646)
[23:55:42] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R2.PlayerConnection.handleCommand(PlayerConnection.java:1133)
[23:55:42] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R2.PlayerConnection.a(PlayerConnection.java:968)
[23:55:42] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R2.PacketPlayInChat.a(PacketPlayInChat.java:45)
[23:55:42] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R2.PacketPlayInChat.a(PacketPlayInChat.java:1)
[23:55:42] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R2.PlayerConnectionUtils$1.run(SourceFile:13)
[23:55:42] [Server thread/WARN]: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[23:55:42] [Server thread/WARN]: 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[23:55:42] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R2.SystemUtils.a(SourceFile:60)
[23:55:42] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R2.MinecraftServer.A(MinecraftServer.java:710)
[23:55:42] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R2.DedicatedServer.A(DedicatedServer.java:368)
[23:55:42] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R2.MinecraftServer.z(MinecraftServer.java:651)
[23:55:42] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R2.MinecraftServer.run(MinecraftServer.java:554)
[23:55:42] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:745)
[23:55:47] [Server thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_40]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_40]
	at net.minecraft.server.v1_8_R2.SystemUtils.a(SourceFile:61) [spigot.jar:git-Spigot-ea179b3-6e0120a]
	at net.minecraft.server.v1_8_R2.MinecraftServer.A(MinecraftServer.java:710) [spigot.jar:git-Spigot-ea179b3-6e0120a]
	at net.minecraft.server.v1_8_R2.DedicatedServer.A(DedicatedServer.java:368) [spigot.jar:git-Spigot-ea179b3-6e0120a]
	at net.minecraft.server.v1_8_R2.MinecraftServer.z(MinecraftServer.java:651) [spigot.jar:git-Spigot-ea179b3-6e0120a]
	at net.minecraft.server.v1_8_R2.MinecraftServer.run(MinecraftServer.java:554) [spigot.jar:git-Spigot-ea179b3-6e0120a]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_40]
Caused by: java.lang.NullPointerException
	at net.minecraft.server.v1_8_R2.ContainerAnvil$2.a(ContainerAnvil.java:68) ~[spigot.jar:git-Spigot-ea179b3-6e0120a]
	at net.minecraft.server.v1_8_R2.Container.clickItem(Container.java:305) ~[spigot.jar:git-Spigot-ea179b3-6e0120a]
	at net.minecraft.server.v1_8_R2.PlayerConnection.a(PlayerConnection.java:1610) ~[spigot.jar:git-Spigot-ea179b3-6e0120a]
	at net.minecraft.server.v1_8_R2.PacketPlayInWindowClick.a(SourceFile:31) ~[spigot.jar:git-Spigot-ea179b3-6e0120a]
	at net.minecraft.server.v1_8_R2.PacketPlayInWindowClick.a(SourceFile:9) ~[spigot.jar:git-Spigot-ea179b3-6e0120a]
	at net.minecraft.server.v1_8_R2.PlayerConnectionUtils$1.run(SourceFile:13) ~[spigot.jar:git-Spigot-ea179b3-6e0120a]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_40]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_40]
	at net.minecraft.server.v1_8_R2.SystemUtils.a(SourceFile:60) ~[spigot.jar:git-Spigot-ea179b3-6e0120a]
	... 5 more
```

**Comment by Siguza** [comment]
I know, working on it.

**Comment by Siguza** [comment]
Fixed with 2168801d9437219876f6220cdd9166e61ce35642.

**Comment by mibby** [comment]
:D Thanks.
